### PR TITLE
Coq 8.9.0

### PR DESCRIPTION
**Pull request by mattam82**
This is a fix to PR #13294 applying a patch to avoid failure of compilation on macOS and fixing the opam file to be in sync with coq's 8.9.dev opam file.

### CODE REVIEW

**Comment by mseri** [comment]
The CI installs and uninstalls one after the other all the packages, on osx by the time the env is setup we have already lost some precious time, when the system tries to install coqide (after having installed and removed successfully coq), we run out of time during the depext step. I think we cannot do much about that. I have tried on my mac and it seems to work, I guess we'll make that be enough

**Comment by mattam82** [comment]
Thanks. I get it now, I hadn’t seen the "simulated" word

File: packages/coqide/coqide.8.9.0/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, University Paris Sud, University Paris 7, Ecole Polytechnique."
```
[the remark this answers is not in the document]
Comment (mattam82) [reply packages/coqide/coqide.8.9.0/opam]: Yes, "and contributors" is more accurate :)

[the remark this answers is not in the document]
Comment (RalfJung) [reply packages/coqide/coqide.8.9.0/opam]: Okay, will you prepare a PR to fix this or should I?

[the remark this answers is not in the document]
Comment (mattam82) [reply packages/coqide/coqide.8.9.0/opam]: Go ahead, sorry I missed that one.

**Pull request merged**